soc/v2/nuvoton/numicro/soc.yml

```diff
@@ -0,0 +1,4 @@
+series:
```

Comment:
should this describe the family `numicro` ?

Reply:
fixed